src/server/db/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "development": {
+    "username": "user",
+    "password": "password",
+    "database": "leket",
+    "host": "localhost",
+    "port": 5433,
+    "dialect": "postgres"
+  },
+  "test": {
+    "username": "root",
+    "password": null,
+    "database": "leket",
+    "host": "192.168.99.100",
+    "port": 5433,
+    "dialect": "postgres"
+  },
+  "production": {
+    "use_env_variable": "DATABASE_URL"
```

Comment:
in terms of code relating to this it won't matter very much since it will still be required and return a json. What is the difference in exporting the json and explicitly supplying the json?